pkg/virt-controller/watch/vmi.go

```diff
@@ -2220,22 +2220,29 @@ func (c *VMIController) updateInterfaceStatus(vmi *virtv1.VirtualMachineInstance
 	indexedVMIStatus := indexIfaceStatus(originalVMIStatus)
 
 	var newStatus []virtv1.VirtualMachineInstanceNetworkInterface
+	for i := range originalVMIStatus {
+		newStatus = append(newStatus, originalVMIStatus[i])
+	}
```

Comment:
Can't you just reuse the `originalVMIStatus`? Then it will be crystal clear that here we only add `interfaces` that are not yet present.

Reply:
Done. @xpivarc

Reply:
Thanks. Ed have some comments so leave the lgtm up to him